### ISSUE TITLE
TripletNetwork overfits due to missing data shuffling during training

### ISSUE THREAD

**Issue by VecherVhatuX** [VecherVhatuX/learning_data_science_in_prison#1388]
The current implementation of the TripletNetwork suffers from overfitting due to the lack of data shuffling during training, and the `DataLoader` initialization has `shuffle=False`, which prevents the model from learning diverse patterns in the data.
